Example/WalletApp/PresentationLayer/Wallet/ConnectionDetails/ConnectionDetailsView.swift

```diff
@@ -0,0 +1,136 @@
+                        AsyncImage(url: URL(string: presenter.session.peer.icons.first ?? "")) { phase in
+                            if let image = phase.image {
+                                image
+                                    .resizable()
+                                    .frame(width: 60, height: 60)
+                                    .background(Color.black)
+                                    .cornerRadius(30, corners: .allCorners)
+                            } else {
+                                Color.black
+                                    .frame(width: 60, height: 60)
+                                    .cornerRadius(30, corners: .allCorners)
+                            }
+                        }
+                        .padding(.bottom, 6)
+                        
+                        Text(presenter.session.peer.name)
+                            .foregroundColor(.grey8)
+                            .font(.system(size: 22, weight: .bold, design: .rounded))
+                        
+                        Text(presenter.session.peer.url)
+                            .foregroundColor(.grey50)
+                            .font(.system(size: 13, weight: .medium, design: .rounded))
+                    }
+                    
+                    ForEach(presenter.session.namespaces.keys.sorted(), id: \.self) { namespace in
+                        VStack {
+                            VStack(alignment: .leading) {
+                                Text(namespace)
+                                    .font(.system(size: 15, weight: .semibold, design: .rounded))
+                                    .foregroundColor(.whiteBackground)
+                                    .padding(.horizontal, 8)
+                                    .padding(.vertical, 5)
+                                    .background(Color.grey70)
+                                    .cornerRadius(28, corners: .allCorners)
+                                    .padding(.leading, 15)
+                                    .padding(.top, 9)
+                                
+                                VStack(spacing: 0) {
+                                    HStack {
+                                        Text("Accounts")
+                                            .foregroundColor(.grey50)
+                                            .font(.system(size: 13, weight: .semibold, design: .rounded))
+                                        
+                                        Spacer()
+                                    }
+                                    .padding(.horizontal, 18)
+                                    .padding(.top, 10)
+                                    
+                                    HStack(spacing: 0) {
+                                        Text("\(presenter.session.namespaces[namespace]?.accounts.first?.namespace ?? "") : \((presenter.session.namespaces[namespace]?.accounts.first?.reference ?? ""))")
```

Comment:
I thing that kind of complex things is better put into presenter: `presenter.namespace`